src/commands.rs

```diff
@@ -250,16 +251,40 @@ impl AssetCommand {
         Ok(Assets::new(get(&self.authentication, &endpoint)?))
     }
 
+    fn add_web_asset(&self, url: &str, headers: &HeaderMap, payload: &HashMap<&str, &String>) -> anyhow::Result<Assets, CommandError> {
+        let response = self
+            .authentication
+            .build_client()?
+            .post(url)
+            .json(payload)
+            .headers(headers.clone())
+            .send()?;
+
+        if response.status().as_u16() != 201 {
```

Comment:
You should use https://docs.rs/reqwest/0.9.21/reqwest/struct.StatusCode.html

Reply:
what's the difference? I don't see it